react/components/LogoutCustomerSession.tsx

```diff
@@ -49,6 +53,26 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm();
+
+  const { id } = orderForm;
+
+  const [customerClass, setCustomerClass] = useState("");
+
+  useEffect(()=> {
+    function getCustomerClass() {
+      var requestOptions = {
+        method: 'GET'
+      }
+      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+    }
+  
+    getCustomerClass().then(response => response.json()).then(data => {
+      if(!!data.clientProfileData && !!data.clientProfileData.customerClass) {
```

Comment:
Do you think this suggestion could be enough?
```suggestion
      if(!!data?.clientProfileData?.customerClass) {
```

Reply:
@lariciamota Updated the code with suggested changes

Reply:
@Suraj-Rajmane Hi! Could you revert this change, please? It made the app test fail:

>    SyntaxError: /github/workspace/react/components/LogoutCustomerSession.tsx: Support for the experimental syntax 'optionalChaining' isn't currently enabled 

We would have to update the config so it could pass, so for simplicity's sake, I'd ask you to discard my suggestion 🙏